pages/services/kubernetes/1.0.0-1.9.3/upgrade/index.md

```diff
@@ -0,0 +1,167 @@
+
+
+## Before you start
+
+Currently, updating the package may trigger an update of a subset or
+all the components managed by this framework, depending on whether new versions
+of these components or related ones are part of the new release.
+
+Updating these components means the Kubernetes cluster may incur in some
+downtime or, in the worst-case scenario, cease to function properly. And while
+we are committed to make the user experience as robust and smooth as possible,
+fact is there is no such thing as fail-proof software.
+
+Before updating the package version, **proceed cautiously and always backup your data**.
+
+In the future, we will be integrating the disaster-recovery functionality so
+the user can easily revert to the previous known-to-work state.
+
+### Updating package vs updating package options
+
+When a new package is available, the user will have the option to update to
+the new version. If the user hasn't updated the package in a while, it may be
+that an update to the most recent package version is not allowed. If this is
+the case, the user will be informed on how to proceed.
+
+However, the user may simply want to update package options, for example, in
+order to change the resources allocated to one type of Kubernetes component.
+Do have in mind that package options updates may also mean the Kubernetes
+cluster may incur in some downtime or, in the worst-case scenario, cease to
+function properly.
+
+Before updating package options, **proceed cautiously and always backup your data**.
+
+## Updating
+
+In order to update the package, the `dcos kubernetes update` subcommand
+is available.
+
+```shell
+$ dcos kubernetes update -h
+usage: dcos kubernetes [<flags>] update [<flags>]
+
+Flags:
+  -h, --help               Show context-sensitive help.
+  -v, --verbose            Enable extra logging of requests/responses
+      --name="kubernetes"  Name of the service instance to query
+      --options=OPTIONS    Path to a JSON file containing the target package options
+      --package-version=PACKAGE-VERSION
+                           The target package version
+      --yes                Do not ask for confirmation before starting the update process
+      --timeout=1200s      Maximum time to wait for the update process to complete
+
+```
+
+**IMPORTANT:** If the user has a cluster with many Kubernetes nodes, we
+recommend adjusting the `--timeout` value to a more generous number but we can't
+calculate in advance what the value should be, given the amount of variables
+involved, such as the DC/OS cluster resources available, CPU and internet speed,
+etc.
+
+### Updating the package version
+
+Below is how the user starts the package version update:
+
+```shell
+$ dcos kubernetes update --package-version=<NEW_VERSION>
+About to start an update from version <CURRENT_VERSION> to <NEW_VERSION>
+
+Updating these components means the Kubernetes cluster may incur in some
```

Comment:
same

Reply:
oh this is in the CLI output... hmmm

Reply:
It's not I don't think - updated to read better anyway, we can fix up in future releases if CLI output is rubbish.

Done.

Reply:
CLI taken care of: https://github.com/mesosphere/dcos-kubernetes/pull/453